build.sbt

```
@@ -95,43 +78,17 @@ addCommandAlias("mima", "coreJVM/mimaReportBinaryIssues")
 addCommandAlias("fmt", "all scalafmtSbt scalafmtAll")
 addCommandAlias("fmtCheck", "all scalafmtSbtCheck scalafmtCheckAll")
 
-lazy val crossVersionSharedSources: Seq[Setting[_]] = Seq(Compile, Test).map { sc =>
-  (sc / unmanagedSourceDirectories) ++= (sc / unmanagedSourceDirectories).value.map { dir: File =>
-    new File(dir.getPath + "-" + scalaBinaryVersion.value.head)
-  }
-}
-
-lazy val publishSettings = Seq(
-  Test / publishArtifact := false,
-  pomIncludeRepository := (_ => false),
-  homepage := Some(url("https://github.com/typelevel/kittens")),
-  licenses := Seq("Apache 2" -> url("http://www.apache.org/licenses/LICENSE-2.0.txt")),
-  scmInfo := Some(ScmInfo(url("https://github.com/typelevel/kittens"), "scm:git:git@github.com:typelevel/kittens.git")),
-  developers := List(
-    Developer("milessabin", "Miles Sabin", "", url("http://milessabin.com/blog")),
-    Developer("kailuowang", "Kai(luo) Wang", "kailuo.wang@gmail.com", url("http://kailuowang.com/")),
-    Developer("joroKr21", "Georgi Krastev", "joro.kr.21@gmail.com", url("https://twitter.com/Joro_Kr"))
-  )
+ThisBuild / licenses := Seq(License.Apache2)
+ThisBuild / developers := List(
+  Developer("milessabin", "Miles Sabin", "", url("http://milessabin.com/blog")),
+  Developer("kailuowang", "Kai(luo) Wang", "kailuo.wang@gmail.com", url("http://kailuowang.com/")),
+  Developer("joroKr21", "Georgi Krastev", "joro.kr.21@gmail.com", url("https://twitter.com/Joro_Kr"))
 )
 
-lazy val noPublishSettings =
-  publish / skip := true
-
-ThisBuild / githubWorkflowJavaVersions := Seq("adopt@1.8")
-ThisBuild / githubWorkflowArtifactUpload := false
-ThisBuild / githubWorkflowBuildMatrixAdditions += "ci" -> List("test")
-ThisBuild / githubWorkflowBuild := List(WorkflowStep.Sbt(List("${{ matrix.ci }}"), name = Some("Validation")))
-ThisBuild / githubWorkflowTargetTags ++= Seq("v*")
-ThisBuild / githubWorkflowPublishTargetBranches := Seq(RefPredicate.StartsWith(Ref.Tag("v")))
-ThisBuild / githubWorkflowPublishPreamble += WorkflowStep.Use(UseRef.Public("olafurpg", "setup-gpg", "v3"))
-ThisBuild / githubWorkflowPublish := Seq(
+ThisBuild / tlCiReleaseBranches := Seq("dotty")
+ThisBuild / githubWorkflowBuild ~= { steps =>
   WorkflowStep.Sbt(
-    List("ci-release"),
-    env = Map(
-      "PGP_PASSPHRASE" -> "${{ secrets.PGP_PASSPHRASE }}",
-      "PGP_SECRET" -> "${{ secrets.PGP_SECRET }}",
-      "SONATYPE_PASSWORD" -> "${{ secrets.SONATYPE_PASSWORD }}",
-      "SONATYPE_USERNAME" -> "${{ secrets.SONATYPE_USERNAME }}"
-    )
-  )
-)
+    List("scalafmtCheckAll", "scalafmtSbtCheck"),
```

This is my mistake on `master` as well. Because commands in a cross-platform-build only run in the scope of `rootJVM` or `rootJS`, none of those actually includes `build.sbt` at the root.

```suggestion
    List("scalafmtCheckAll", "root/scalafmtSbtCheck"),
```